server/demo/demoresource/demo_resource.go

```diff
@@ -0,0 +1,344 @@
+package demoresource
```

Comment:
it's not neccesary to have a package for each resource. This could live in `config/demoresource`. Not neccesary but maybe it's not great to have a bunch of almost empty packages? More of a question than a suggestion

Reply:
makes sense to have it on the config package.